jscripts/bbcodes_sceditor.js

```diff
@@ -174,7 +174,7 @@ $(function ($) {
 
 	$.sceditor.command.set('size', {
 		_dropDown: function (editor, caller, callback) {
-			var content = $('<div />'),
+			var content = $('<div></div>'),
```

Comment:
Were these broken after the update?

Reply:
This is the main change that needs to be done with the update.

Reply:
Looks like singular elements should be fine (and can be found in more places in core and dependencies): https://jquery.com/upgrade-guide/3.5/#description-of-the-change

Reply:
Closing tag is required to be declared separately now. There was a bug running in DOM while auto-closing tag parsing was in place, and the feature is dropped from jQuery 3.5.0 +

I have searched the entire codebase but couldn't find any such implementation that requires attention. Appreciate others to try detecting any existing ...

Reply:
Is this a different case? https://github.com/mybb/mybb/blob/mybb_1822/jscripts/usercp.js#L55

Reply:
Nope, this should also be addressed in similar way.
I'll do it shortly.

Reply:
What happens when it's not changed? I don't see any messages or broken functionality on 3.5.1.

Reply:
It breaks the HTML structure.
Try for the editor code / PHP changes and see what happens in editor.

Note, I have found few more, in acp js that I will fix too.

Reply:
The `<textarea>` corrections seem necessary since they're nested, but the simple ones like `$('<div />')` might be fine.

Reply:
May be some DOM / borwser side correction is correcting the same and you are not experiencing the issue but if you go through the "Description of the change" its clearly stated that not to use such shorthand for closing.

You were right about the nesting as `<div /></span>` now produces `<div><span></span></div>` in place of earlier `<div></div><span></span>` but at the end of the day jQuery.htmlPrefilter closes the tags and make valid HTML. Wherever we have used such shorthands I have changed to open-close just to avoid any unnecessary nesting issues and manually declaring closing tag is completely valid and acceptable to jQuery.

Now suggest me how you want it to be done and I'll modify accordingly.

Reply:
> [...] don't use self-closing tags for tags that may have content [...]

It sounds like we'll be fine with these, since they have no content (it's also stated that the `jQuery()` call still transforms tags and closes them).

Even if closing them is best coding practice, it's probably better off with no unnecessary changes, especially if there are no errors or deprecation notices.

Reply:
OK, I'm reverting that change shortly.